test/src/state_machine/dummy.test.cpp

```diff
@@ -0,0 +1,33 @@
+#include "gtest/gtest.h"
+#include "utils/logger.hpp"
+#include "data/data.hpp"
+#include "state_machine/hyped-machine.hpp"
+
+  using hyped::data::Data;
+  void t() {
+    Data& d = Data::getInstance();
+  }
+  TEST(dummy_test, f) {
+    //Data &d = Data::getInstance();
+    ASSERT_EQ(2,1+1);
+    // ASSERT_EQ(3,1+1);
+
```

Comment:
Empty line??

Reply:
We are not really linting the test sources at the moment. I want to create a subtask to take care of this later. Do you insist this has to be done in this PR?

Reply:
new line added